peer/configuration.proto

```diff
@@ -32,3 +32,9 @@ message APIResource {
 message ACLs {
     map<string, APIResource> acls = 1;
 }
+
+// PrivateDataImplicitCollectionConfig stores the configuration for org-specific implicit collections
+message PrivateDataImplicitCollectionConfig {
```

Comment:
Not to bikeshed, but this stutters a bit.  We don't include `Config` in any of the other names.

It's `MSP`, not `MSPConfig`, it's `AnchorPeers` not `AnchorPeersConfig`, `BatchSize`, not `BatchSizeConfig` etc.

Maybe just `PrivateDataImplicitCollection` ?

Reply:
I have no preference, @denyeart suggested the name originally. Thoughts?

Reply:
Dave agreed, pushed